src/calculateRentalCost.js

```diff
@@ -3,8 +3,18 @@
  *
  * @return {number}
  */
+
+const seven = 7;
+const three = 3;
+
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= seven) {
+    return days * 40 - 50;
```

Comment:
Avoid using literals in expressions (HARDCODE). Instead use constants explaining these values. For example, you can use `DAILY_RENTAL_COST` and `LONG_TERM_DISCOUNT`.